src/System.Net.WebSockets.Client/src/Resources/Strings.resx

```diff
@@ -121,4 +121,10 @@
   <data name="net_securityprotocolnotsupported" xml:space="preserve">
     <value>The requested security protocol is not supported.</value>
   </data>
+  <data name="net_WebSockets_UWPClientCertSupportRequiresWindows10InsiderPreviewBuild16215OrGreater" xml:space="preserve">
+    <value>Support for client certificates in UWP requires Windows 10 Insider Preview Build 16215 or greater.</value>
```

Comment:
We do this for now since this is the official pre-release name and version.  When the Windows 10 Fall Creators Update ships, we'll update this again with the official RTM Windows version name. This is similar to what we do with "Windows 10 Version 1607" which means build 14393.